### PR TITLE
add compound seq to custom params and remove

### PR DESCRIPTION
**Pull request by nhart**

### CODE REVIEW

File: islandora_transforms/RELS-EXT_to_solr.xslt [hunk 1]
```diff
@@ -58,14 +58,17 @@
...
-      <xsl:call-template name="rels_ext_fields">
-        <xsl:with-param name="prefix" select="$prefix"/>
-        <xsl:with-param name="suffix" select="$suffix"/>
-        <xsl:with-param name="type">literal</xsl:with-param>
-        <xsl:with-param name="value" select="text()"/>
-      </xsl:call-template>
-    </xsl:template>
+      <!-- XXX: Parameters as passed in are strings... Let's deal with it as a
+        string here, for convenience. -->
+      <xsl:if test="string($index_compound_sequence) = 'true' or (string($index_compound_sequence) = 'false' and (not(self::islandora-rels-ext:isSequenceNumberOf) and  not(self::islandora-rels-ext:* and starts-with(local-name(), 'isSequenceNumberOf')))">
```
Comment (adam-vessey) [inline islandora_transforms/RELS-EXT_to_solr.xslt]: I don't believe the first bit should ever have an effect... the test for `self::islandora-rels-ext:isSequenceNumberOf`... I don't think we'll ever have a relationship of the like, without the PID bit on the end... which is accounted for with the other bit.
```suggestion
      <xsl:if test="string($index_compound_sequence) = 'true' or (string($index_compound_sequence) = 'false' and not(self::islandora-rels-ext:* and starts-with(local-name(), 'isSequenceNumberOf')))">
```

File: custom_parameters.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+index_ancestors = true
+index_compound_sequence = false
```
Comment (adam-vessey) [inline custom_parameters.properties:2]: I realize this is probably desired in _this_ environment... really, _most_ environments, but... when pulled out for general use, should we make it be `true`, to maintain legacy behaviours? Maybe? Dunno...

Comment (willtp87) [reply custom_parameters.properties:2]: Afraid so.

File: islandora_transforms/RELS-EXT_to_solr.xslt [hunk 3]
```diff
@@ -58,14 +58,17 @@
...
-      <xsl:call-template name="rels_ext_fields">
-        <xsl:with-param name="prefix" select="$prefix"/>
-        <xsl:with-param name="suffix" select="$suffix"/>
-        <xsl:with-param name="type">literal</xsl:with-param>
-        <xsl:with-param name="value" select="text()"/>
-      </xsl:call-template>
-    </xsl:template>
+      <!-- XXX: Parameters as passed in are strings... Let's deal with it as a
+        string here, for convenience. -->
+      <xsl:if test="string($index_compound_sequence) = 'true' or (string($index_compound_sequence) = 'false' and not(self::islandora-rels-ext:* and starts-with(local-name(), 'isSequenceNumberOf')))">
```
Comment (adam-vessey) [inline islandora_transforms/RELS-EXT_to_solr.xslt:64]: `islandora-rels-ext` namespace is undefined... needs a:
```xml
    xmlns:islandora-rels-ext="http://islandora.ca/ontology/relsext#"
```
somewhere above here in the XML tree.

**Review by adam-vessey — CHANGES_REQUESTED** [review]

File: islandora_transforms/RELS-EXT_to_solr.xslt (hunk 3 above)
Comment (adam-vessey) [reply islandora_transforms/RELS-EXT_to_solr.xslt:64]: [Addressed](https://github.com/discoverygarden/cuhk-basic-solr-config/pull/14/commits/66fc93f160b86812a23b8c2637804e8ac5e9ab7e#diff-9823cf446d455734ea3ac88427f23211R8).

File: custom_parameters.properties (hunk 2 above)
Comment (adam-vessey) [reply custom_parameters.properties:2]: Being done across:
* https://github.com/discoverygarden/basic-solr-config/pull/196/commits/059cdf5c43793dfc02d69feaed990c6dfe3db15a
* https://github.com/discoverygarden/basic-solr-config/pull/197/commits/90a6f9cbcd8a253eed97c600d668129f7b7970ed and
* https://github.com/discoverygarden/basic-solr-config/pull/198/commits/0c706d8f0ba67f5c988c82067b0ccdcf916ab4f6

File: foxmlToSolr.xslt [hunk 4]
```diff
@@ -38,6 +38,12 @@
   <xsl:param name="TRUSTSTOREPATH" select="repositoryName"/>
   <xsl:param name="TRUSTSTOREPASS" select="repositoryName"/>
 
+  <!--
+    Parameter(s) from custom_parameters.properties.
+  -->
+  <xsl:param name="index_ancestors" select="false()"/>
+  <xsl:param name="index_compound_sequence" select="false()"/>
```
Comment (adam-vessey) [inline foxmlToSolr.xslt]: Not terribly concerned, but... potential trap here, with the coded default being different from [that being made in core](https://github.com/discoverygarden/basic-solr-config/pull/196/files#diff-3d413d43a0ca821813d2777605442fd3R47)... paranoia?

**Review by adam-vessey — APPROVED** [review]

File: islandora_transforms/RELS-EXT_to_solr.xslt [hunk 5]
```diff
@@ -58,14 +59,17 @@
...
+        <xsl:call-template name="rels_ext_fields">
+          <xsl:with-param name="prefix" select="$prefix"/>
+          <xsl:with-param name="suffix" select="$suffix"/>
+          <xsl:with-param name="type">literal</xsl:with-param>
+          <xsl:with-param name="value" select="text()"/>
+        </xsl:call-template>
+      </xsl:template>
+    </xsl:if>
```
Comment (adam-vessey) [inline islandora_transforms/RELS-EXT_to_solr.xslt:72]: Closing template/if tags in the wrong order, here.

**Pull request merged**